### DISCUSSION TITLE
New big changes or upkeep and maintenance

### DISCUSSION THREAD

**Discussion by swbog** [Jyers/Marlin#176] [Q&A] [answered]
Really enjoying this firmware. Works a treat. I was just wondering as subject line states if you’re planning on any other big changes or is what you envisioned when you began this project pretty much already here and now it’s just getting dialed in.

**Comment by Schnabulation** [comment]
There are some pull requests I am looking forward to see (ex. UBL menu)...

**Comment by Jyers** [comment, marked as answer]
I believe there are still a lot of additions to be made in the effort to push config options to the on screen menu. Such as pid tuning, UBL, and other general toggles. Besides that I don't have any super major features in the works currently, but who knows what the community might think of for PRs and such. I think the next big change will be pushing everything over to MarlinUI so I can start working on getting this pushed upstream
